tutorials/fundamentals/installation.md

```diff
@@ -1,55 +1,106 @@
+      jupytext_version: 1.13.0
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# How to install napari on your machine
+
+Welcome to the **napari** installation guide!
+
+This guide will teach you how to do a clean install of **napari** and launch the viewer.
+
+```{note} 
+If you want to contribute back to the napari codebase, you should follow [this guide instead](https://napari.org/developers/contributing.html).
+```
 
-Welcome to the **napari** installation tutorial!
+## Prerequisites
 
-This tutorial will teach you how to do a clean install of **napari**. It is
-aimed at people that just want to use napari. For people also interested in
-contributing to napari please check our [contributing
-guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
-for more advanced installation procedures. At the end of the tutorial you should
-have napari successfully installed on your computer and be able to make the
-napari viewer appear.
+Prerequisites differ depending on what type of distribution you want to install.
 
-## installation
+### Installing the bundled app
+This is the easiest way to install napari if you only wish to use it as a standalone GUI app.
+This installation method does not have any prerequisites. 
 
-### which distribution to install
+```{important}
+Note that the bundled app is still
+in active development, and may not be very stable. We strongly recommend installing as a Python
+package instead.
+```
 
-If you want to contribute back to the napari codebase, you should install from
-source code: see the [from source](#clone-the-repository-locally-and-install-in-editable-mode) section.
+[Click here](#installing-as-a-bundled-app) to see instructions
+for installing the bundled app.
 
-If you only wish to use napari as GUI app (and not from python code), the
-[bundled app](#installing-as-a-bundled-app)
-is the easiest way to install, it does not require having python pre-installed.
+### Installing as a python package 
+This installation method allows you to use napari from Python to programmatically 
+interact with the app. It is the best way to install napari and make full use of
+all its features.
 
-If you are using napari from Python to programmatically interact with the app,
-you can install via pip, conda-forge, or from source.
+It requires:
+- [Python 3.7+](https://www.python.org/downloads/)
+- the ability to install python packages via [pip](https://pypi.org/project/pip/) OR [conda-forge](https://conda-forge.org/docs/user/introduction.html)
 
-### from pip, with "batteries included"
+You may also want:
+- an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
+[venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
 
-napari can be installed on most macOS, Linux, and Windows systems with Python
-3.7, 3.8, and 3.9 using pip:
+## Choose which distribution to install
```

Comment:
Yeahh this one is a bit weird because "Choose which distribution to install" does only refer to installing as package - it's just do you want it from `pip`, `conda` or `github main branch`. I'll change the wording here to reflect this a bit better.